src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -725,29 +730,142 @@ public function run() {
+      pht('  2. Make your changes and commit them.'),
+      pht('  3. Publish your PR: %s', phutil_console_format('**arh publish**')),
+      pht('  Documentation: %s', phutil_console_format('__%s__', 'https://p.uber.com/arh')),
+      pht('If you must use Phabricator, run: %s', phutil_console_format('**arc diff --force**')),
+      pht('This will prompt you for an exemption reason.')
+    );
+    
+    throw new ArcanistUserAbortException();
+  }
+
+  /**
+   * Prompt user for exemption reason when using --force to create Phabricator diff.
+   * Only prompts users in the GitHub beta group.
+   * Shows GitHub invitation banner first as a "last chance" invitation.
    */
-  private function displayGitHubInvitationBanner() {
+  private function promptForForceExemptionReason() {
+    // Skip prompting in non-interactive scenarios
+    if (!$this->isTTY()) {
+      return;
+    }
+
+    // Only prompt GitHub beta group users for exemption reason
     $gbu = new UberGitHubBetaUsersPrompt();
-    $isMember = $gbu->isCurrentUserInGitHubBetaUsersPromptGroup();
-    if (!$isMember) {
+    if (!$gbu->isCurrentUserInGitHubBetaUsersPromptGroup()) {
       return;
     }
 
-    // Check if current repository is in the deny list for GitHub PR prompts
+    // Show GitHub invitation banner first as a "last chance" invitation
+    $this->displayGitHubInvitationBannerOnly();
+
+    $reason_prompt = pht(
+      'You are forcing a Phabricator diff instead of using GitHub PR. Please explain why:'
+    );
+    
+    $reason = phutil_console_prompt($reason_prompt);
+    if (empty(trim($reason))) {
+      $this->console->writeErr(
+        "%s\n",
+        pht('An exemption reason is required when using --force.')
+      );
+      throw new ArcanistUserAbortException();
+    }
+    
+    // Track the exemption reason
+    $this->trackForceExemptionReason(trim($reason));
+    
+    $this->console->writeOut(
+      "%s: %s\n\n", 
+      pht('Exemption reason recorded'),
+      trim($reason)
+    );
+  }
+
+  /**
+   * Display just the GitHub invitation banner without any prompts.
+   * Used for --force users as a "last chance" invitation.
+   */
+  private function displayGitHubInvitationBannerOnly() {
+    // Check if repository is disabled for GitHub prompts
     if ($this->isRepositoryDisabledForGitHubPrompt()) {
       return;
     }
     
+    $this->outputGitHubInvitationBanner();
+  }
+
+  /**
+   * Output the GitHub invitation banner content.
+   */
+  private function outputGitHubInvitationBanner() {
     $docs_link = "\033]8;;https://p.uber.com/arh\033\\p.uber.com/arh\033]8;;\033\\";
     $slack_link = "\033]8;;https://uber.enterprise.slack.com/archives/C05H81RFE4F\033\\#github-pr-beta\033]8;;\033\\";
 
     $banner = <<<EOBANNER
 **************************************
 *                                    *
-*  You're invited to try GitHub!     *
+*  Please consider using GitHub!     *
```

Comment:
```suggestion
*  Uber is moving to GitHub!     *
```

^ Since we're moving towards telling people vs asking them, asking `Please consider` feels inaccurate now. Suggest updating the language to reflect the shift in tone.